backend/src/test/scala/cromwell/backend/validation/RuntimeAttributesValidationSpec.scala

```diff
@@ -278,5 +282,81 @@ class RuntimeAttributesValidationSpec extends WordSpecLike with Matchers with Be
         case Invalid(e) => assert(e.head == "Failed to get cpu mandatory key from runtime attributes")
       }
     }
+    
+    "contain expected default values based on a config file" in {
+      val defaultVals = Map(
+        "cpu" -> CpuValidation.configDefaultWdlValue(TestConfig.allRuntimeAttrsConfig).get,
+        "failOnStderr" -> FailOnStderrValidation.configDefaultWdlValue(TestConfig.allRuntimeAttrsConfig).get,
+        "continueOnReturnCode" -> ContinueOnReturnCodeValidation.configDefaultWdlValue(TestConfig.allRuntimeAttrsConfig).get
+      )
+
+      val expectedDefaultVals = Map(
+        "cpu" -> WdlInteger(1),
+        "failOnStderr" -> WdlBoolean(false),
+        "continueOnReturnCode" -> WdlInteger(0)
+      )
+
+      defaultVals shouldBe expectedDefaultVals
+    }
+
+    "return default values as WdlString when can't be coerced to valid WdlType" in {
```

Comment:
The reason they are pushed forward because the VRAB does a beautiful job of listing all problems with the runtime attributes at once. If I failed at this stage of the evaluation, it would be one attribute at a time, which sounds frustrating from a user's perspective. We know that the un-coercable values won't be used by any backend as the VRAB makes sure of that, is there a risk otherwise?